### PR TITLE
WT-10582 Update gcp providers in test/evergreen to use WiredTiger production accounts

### PR DESCRIPTION
**Pull request by jiechenbo**
Previously we have disabled gcp tests because the credentials we previously had expired due to using my personal account. Since then we have generated production accounts and we can now re-enable back the tests.

### CODE REVIEW

File: ext/storage_sources/gcp_store/test/test_gcp_connection.cpp [hunk 1]
```diff
@@ -94,7 +94,7 @@ generate_test_prefix()
 
 TEST_CASE("Testing class gcpConnection", "gcp-connection")
 {
-    std::string test_bucket_name = "unit_testing_gcp";
+    std::string test_bucket_name = "gcptestext-us";
```
Comment (ajmorton) [inline ext/storage_sources/gcp_store/test/test_gcp_connection.cpp:97]: From memory we delete the contents of these buckets on test finish, so having the python and unittests run on the same bucket in parallel could cause interference. Were there issues with creating a new unit_testing_gcp bucket?

**Review by ajmorton — COMMENTED** [review]
One question. I've also enabled all of the tiered tests for this PR to be safe.

File: ext/storage_sources/gcp_store/test/test_gcp_connection.cpp (hunk 1 above)
Comment (jiechenbo) [reply ext/storage_sources/gcp_store/test/test_gcp_connection.cpp:97]: We delete all of the unit test related contents to do with the test run. We loop through all objects of the unit test through a generated prefix through the generate_test_prefix() function. 
Example of generated test prefix: "gcptest/unit/2022-31-01-16-34-10/623843294/".

**Review by ajmorton — APPROVED** [review]
LGTM. Thanks!

**Comment by jiechenbo** [comment]
evergreen merge

**Pull request merged**